build.gradle

```diff
@@ -149,6 +150,7 @@ subprojects {
 			spring_boot_starter_web          : "org.springframework.boot:spring-boot-starter-web:${springWebVersion}",
 			spring_cloud_starter_openfeign   : "org.springframework.cloud:spring-cloud-starter-openfeign:${springOpenFeignVersion}",
 			spring_cloud_sleuth_otel_autoconf: "org.springframework.cloud:spring-cloud-sleuth-otel-autoconfigure:${springOtelVersion}",
+			opencensus_shim: "io.opentelemetry:opentelemetry-opencensus-shim:${opencensusShim}-alpha"
```

Comment:
After this change, our tests always (rather than never) have the OC shim on the classpath. Can we test both scenarios, e.g. by having a subproject for the sole purpose of testing OC interactions?

@dashpole curious if you think OC shim interactions should be tested here vs in the OC shim.

Reply:
IMO, the OC shim interactions should be tested here since we are making the changes here to fix the issue.

The idea of having a separate project to test the *without shim* scenario would definitely add some value, but it may not be necessary since without OC shim might be the *general case* (I may be wrong here). Do we have a reason to believe that removing the dependency from OC shim would cause any issues ?

Reply:
I have removed this dependency and the unit test that attempted to test the failure case with this dependency. When running as GitHub actions, the tests were behaving differently from their local run. 

Let me know if having such a unit test in exporter-trace is still a good idea. Punya's suggestion of having a subproject which specifies a dependency on both - shim & exporter trace seems like a good place for it, but not sure if the extra effort is worth the gains.

Reply:
This is moot now, since deferred approach is the only approach that is currently available. So we cannot replicate the failure situation now. 

Current unit tests verify that the trace exporter works without any issues **even if** OC-shim is on the classpath.